package ru.example.qa.notesapp.presentation.exception.auth

sealed class AuthException : Throwable()